contracts/kernel/Kernel.sol

```diff
@@ -72,6 +72,23 @@ contract Kernel is IKernel, KernelStorage, Initializable, AppProxyFactory, ACLSy
         return apps[_id];
     }
 
+    /**
+    * @dev Get the address of the default Vault instance (to recover funds)
+    * @return Address of the Vault
+    */
+    function getDefaultVault() public view returns (address) {
+        return apps[defaultVaultId];
+    }
+
+    /**
+    * @dev Set the resolving address of an app instance or base implementation
+    * @param _name Name of the app
+    * @return ID of app
+    */
+    function setDefaultVaultId(bytes32 _name) auth(APP_MANAGER_ROLE, arr(APP_BASES_NAMESPACE, _name)) public {
+        defaultVaultId = keccak256(APP_BASES_NAMESPACE, _name);
```

Comment:
I think we should make this part of the initialization of the kernel. Set this to `defaultVaultId = keccak256(APP_BASES_NAMESPACE, namehash(`vault.aragonpm.eth`)`

Reply:
See #282, as the name should be the appId for the vault

Reply:
That should have been `APP_ADDR_NAMESPACE`